src/SourceGenerators/Uno.UI.SourceGenerators/RemoteControl/RemoteControlGenerator.cs

```diff
@@ -85,6 +88,29 @@ public void Execute(GeneratorExecutionContext context)
 					context.AddSource("RemoteControl", sb.ToString());
 				}
 			}
+			else if (isProjectConfigEnabled || isServerProcessorsConfigEnabled)
+			{
+				// This is designed for runtime-tests that wants to test HotReload also in release (i.e. using app built on the CI).
+				// The runtime-test package will add those properties if the dev-server package is referenced so apps will be automatically
+				// configured properly as soon as they reference both the dev-server and the runtime-test package.
+
+				var sb = new IndentedStringBuilder();
+				BuildGeneratedFileHeader(sb);
+				
+				if (isProjectConfigEnabled)
+				{
+					BuildProjectConfiguration(context, sb);
+				}
+				if (isServerProcessorsConfigEnabled)
+				{
+					BuildServerProcessorsPaths(context, sb);
+				}
+
+				context.AddSource("RemoteControl", sb.ToString());
+			}
+
+			bool Is(string propertyName)
```

Comment:
Let's not be minimalistic :)

```suggestion
			bool GetMSBuildPropertyValue(string propertyName)
```

Reply:
It also checks if set to `true` so it should not be `GetXXX` but `IsXXX`  :p